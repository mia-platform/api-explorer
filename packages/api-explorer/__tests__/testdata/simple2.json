{
    "components": {
        "foo": {
            "type": "object",
            "properties":{
                "fields": {
                    "$ref": "#/components/bar"
                }
            } 
        },
        "bar": {
            "type": "array",
            "items": {
                "type": "string"
            }
        }
        
    },
    "type": "array",
    "items": {
        "$ref": "#/components/foo"
    }
}