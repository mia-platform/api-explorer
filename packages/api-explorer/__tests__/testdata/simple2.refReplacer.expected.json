{
    "definitions": {
        "components": {
            "definitions": {
                "foo": {
                    "type": "object",
                    "properties":{
                        "fields": {
                            "$ref": "#/definitions/components/definitions/bar"
                        }
                    } 
                },
                "bar": {
                    "type": "array",
                    "items": {
                        "type": "string"
                    }
                }
            }

        }
    },
    "type": "array",
    "items": {
        "$ref": "#/definitions/components/definitions/foo"
    }
}